config/zones.json

```diff
@@ -2754,7 +2773,8 @@
       "wind": 4626
     },
     "contributors": [
-      "https://github.com/corradio"
+      "https://github.com/corradio",
```

Comment:
are you sure you didn't mean to update the key "HR"?

Reply:
I meant both if that's ok? What's the criteria?

Reply:
That's completely fine. But don't you also want to add yourself to CR as you update the CR production parser?